File: Backend/API/src/main/java/com/example/localhistory/token/dto/response/TokenDTO.java
package com.example.localhistory.token.dto.response;

import com.example.localhistory.user.model.User;
import jakarta.persistence.*;

import java.time.LocalDateTime;

public class TokenDTO {
    private Integer id;
    private String refreshToken;
    private LocalDateTime expirationDate;
    private boolean isRevoked;
    private LocalDateTime revokedAt;
    private LocalDateTime createdAt;
    private String jwtToken;
    public boolean isExpired;
}

File: Frontend/app/src/main/java/com/example/localhistory/model/response/TokenDTO.kt
package com.example.localhistory.model.response

import kotlinx.datetime.LocalDateTime

data class TokenDTO(
    val id: Int,
    val refreshToken: String,
    val expirationDate: LocalDateTime,
    val isRevoked: Boolean,
    val revokedAt: LocalDateTime?,
    val createdAt: LocalDateTime,
    val jwtToken: String,
    val isExpired: Boolean
)
